pkg/portutil/portutil.go

```diff
@@ -59,6 +59,7 @@ func ParseFlagP(s string) ([]cni.PortMapping, error) {
 	case 2:
 		proto = strings.ToLower(splitBySlash[1])
 		switch proto {
+		// sctp is not a supported protocol
```

Comment:
The comment suggests `SCTP` isn't supported, but it is present in the case line `63`

Reply:
removed